src/char/char.c

```diff
@@ -1587,17 +1587,25 @@ int char_make_new_char_sql(struct char_session_data *sd, const char *name_, int
 	if( sd->found_char[slot] != -1 )
 		return -2; /* character account limit exceeded */
 
+	switch(sex) {
+		case SEX_FEMALE:
```

Comment:
We need a default value too, just in case someone modifies the packet and send it.